src/modules/dashboard/sagas/motions/manageReputationMotion.ts

```diff
@@ -33,23 +33,28 @@ function* smiteMotion({
     amount,
     annotationMessage,
     motionDomainId,
+    isSmitingReputation,
   },
   meta: { id: metaId, history },
   meta,
-}: Action<ActionTypes.COLONY_MOTION_SMITE>) {
+}: Action<ActionTypes.COLONY_MOTION_MANAGE_REPUTATION>) {
   let txChannel;
   try {
     /*
      * Validate the required values
      */
     if (!userAddress) {
-      throw new Error('A user address is required to smite this user');
+      throw new Error(
+        'A user address is required to manage the reputation this user',
+      );
     }
     if (!domainId) {
-      throw new Error('Domain id not set for Smite transaction');
+      throw new Error(
+        'Domain id not set for manage the reputation transaction',
```

Comment:
I am nitpicking but  I would say `Domain id not set for the manage reputation transaction`. Otherwise it sounds a bit off.